src/Employer/Employer.Web/ViewModels/CreateVacancyOptions/CreateVacancyOptionsViewModel.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+using Microsoft.AspNetCore.Mvc.Rendering;
+
+namespace Esfa.Recruit.Employer.Web.ViewModels.CreateVacancyOptions
+{
+    public class CreateVacancyOptionsViewModel 
+    {
+        [Required(ErrorMessage = ValidationMessages.CreateVacancyOptionsConfirmationMessages.SelectionRequired)]
+        public Guid? VacancyId { get; set; }
```

Comment:
We shouldn't be posting a a view model should we?

Reply:
??